cmd/otelcontribcol/go.mod

```diff
@@ -4,9 +4,11 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.21.0
 
-toolchain go1.21.9
+toolchain go1.21.5
```

Comment:
Is this needed?

Reply:
My bad, definitely not needed. Thanks for catching that, I've reverted this.